include/ceed.h

```diff
@@ -277,53 +277,41 @@ typedef enum {
 
 CEED_EXTERN const char *const CeedTransposeModes[];
 
-/// Denotes whether a L-vector is ordered [component, node] or [node, component]
-///   with the right-most index being contiguous in memory
-/// @ingroup CeedElemRestriction
-typedef enum {
-  /// L-vector data is not interlaced, ordered [component, node]
-  CEED_NONINTERLACED,
-  /// L-vector data is interlaced, ordered [node, component]
-  CEED_INTERLACED
-} CeedInterlaceMode;
-
-CEED_EXTERN const char *const CeedInterlaceModes[];
-
 /// Argument for CeedElemRestrictionCreateStrided that L-vector is in
 /// the Ceed backend's preferred layout. This argument should only be used
 /// with vectors created by a Ceed backend.
 /// @ingroup CeedElemRestriction
 CEED_EXTERN const CeedInt CEED_STRIDES_BACKEND[3];
 
-CEED_EXTERN int CeedElemRestrictionCreate(Ceed ceed, CeedInterlaceMode imode,
-    CeedInt nelem, CeedInt elemsize, CeedInt nnodes, CeedInt ncomp,
-    CeedMemType mtype, CeedCopyMode cmode, const CeedInt *indices,
+CEED_EXTERN int CeedElemRestrictionCreate(Ceed ceed, CeedInt nelem,
+    CeedInt elemsize, CeedInt ncomp, CeedInt compstride, CeedInt lsize,
+    CeedMemType mtype, CeedCopyMode cmode, const CeedInt *offsets,
```

Comment:
I kept `nnodes` but renamed and rescaled it as `lsize`. It is useful for `CeedElemRestrictionGetMultiplicity()`.

Reply:
That seems fine to me.